src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/projects/ProjectSampleMetadataImportPage.java

```diff
@@ -55,4 +58,15 @@ public int getFoundCount() {
 	public int getMissingCount() {
 		return Integer.parseInt(missingPill.findElement(By.className("badge")).getText());
 	}
+
+	public List<String> getNumberColumnValues() {
```

Comment:
This method is a bit too specific to the test data being used in `ProjectSampleMetadataImportPageIT.java`.  Can you change it to accept the header name as a string so it can be passed in by the test?  That way we can better reuse this page object if we wanted different input data.

Reply:
Updated